lib/matplotlib/tests/test_font_manager.py

```diff
@@ -131,7 +131,33 @@ def test_find_ttc():
         fig.savefig(BytesIO(), format="ps")
 
 
-def test_user_fonts():
+@pytest.mark.skipif(sys.platform != 'linux', reason='Linux only')
+def test_user_fonts_linux(tmpdir, monkeypatch):
+    font_test_file = 'mpltest.ttf'
+
+    # Precondition: the test font should not be available
+    fonts = findSystemFonts()
+    assert not any(font_test_file in font for font in fonts)
```

Comment:
This is only to make sure that the test can run. It’s not part of the actual test condition. Shouldn‘t this better be a pytest.fail or pytest.skip?

Reply:
Hmm, possibly. I just copied this test from the one below.

Reply:
OK, I've added a skip if this condition isn't met.